pallets/file-system/src/lib.rs

```diff
@@ -131,13 +132,10 @@ pub mod pallet {
             + CheckedAdd
             + CheckedSub
             + PartialOrd
-            + FixedPointNumber;
+            + Zero;
 
         /// The type to convert a threshold to a block number.
-        type ThresholdTypeToBlockNumber: Convert<Self::ThresholdType, BlockNumberFor<Self>>;
-
-        /// The type to convert a block number to a threshold.
-        type BlockNumberToThresholdType: Convert<BlockNumberFor<Self>, Self::ThresholdType>;
+        type ThresholdTypeToBlockNumber: ConvertBack<Self::ThresholdType, BlockNumberFor<Self>>;
```

Comment:
Why do we implement the Convert trait for this one in the mock when here we only need ConvertBack? Maybe a better solution would be to have a Convert from BlockNumber to ThresholdType instead

Reply:
We use both `convert` and `convert_back` in file system pallet.

Reply:
Oh but we only require the ConvertBack trait in the config of the pallet, how does that work?

Reply:
Oh it's because the `ConvertBack` inherits `Convert`. So you don't have to specify both.